packages/node/src/utils.ts

```diff
@@ -73,9 +75,54 @@ export enum EdgeRuntimes {
   ExperimentalEdge = 'experimental-edge',
 }
 
+export const IsomorphicRuntime = 'nodejs';
```

Comment:
Perhaps this should be 

```js
export const NodejsRuntimes = [
  LegacyNodejs = undefined,
  IsomorphicNodejs = 'nodejs',
]
```

Reply:
Typescript does not accept `undefined` in a string enum, but I agree it's better code-as-documentation.
I've relaxed the compiler and included it.